code/game/objects/items/clown_items.dm

```diff
@@ -80,6 +80,11 @@
 	cleanspeed = 3 //Only the truest of mind soul and body get one of these
 	uses = 301
 
+/obj/item/soap/omega/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is using [src] to clean themselves from the timeline! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	new /obj/structure/chrono_field(user.loc, user)
+	return

```

Comment:
i thought for suicides this might be good for readability but if you think this is a worthless trailing return i can remove it